book/02-using-atom/sections/06-customizing.asc

```diff
@@ -217,3 +218,18 @@ The scope name is shown in the settings view for each language. Search for the l
 
 .Finding a language grammar
 image::images/python-grammar.png[python grammar]
+
+[[_customizing_language_recognition]]
+==== Customizing Language Recognition
+
+If you want to customize what language package is used when loading a file of a certain extension, Atom can be configured to do this. For now, you'll need to manually edit your Atom `config.cson` file. You can open it using the _Application: Open Your Config_ command from the Command Palette. For example, if you wanted to add the `foo` extension to the CoffeeScript language, you could add this to your configuration file under the `*.core` section:
```

Comment:
What do you think about something like:

`If you want to customize which language package is used when loading files with a certain extension, you need only manually edit your Atom `config.cson` file.`